riotfile.py

```diff
@@ -909,6 +909,18 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
             ],
         ),
+        Venv(
+            name="django_celery",
+            command="pytest {cmdargs} tests/contrib/django_celery",
+            pkgs={
+                "django": "==2.2.1",
```

Comment:
is this venv specifically for a regression only?

we should probably have a comment on why we are only testing this very specific version.

otherwise we should expand this test matrix to be more exhaustive